sdk/core/core/inc/az_platform.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PLATFORM_H
+#define _az_PLATFORM_H
+
+#include <az_result.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <_az_cfg_prefix.h>
+
+/**
+ * @brief az_platform_clock_msec gets platform clock in milliseconds.
```

Comment:
Do not re-state the method name within the docs and make sure they are complete sentences.
```suggestion
 * @brief Gets the platform clock in milliseconds.
```

Reply:
Oh, we do it this way now? That's great, it is even better. I took a comment from another file as a template, we should fix it too.